download/Cargo.toml

```diff
@@ -23,6 +23,8 @@ thiserror.workspace = true
 url.workspace = true
 
 [dev-dependencies]
-hyper = { version = "0.14", default-features = false, features = ["tcp", "server"] }
+http-body-util = "0.1.0"
```

Comment:
This change should be squashed into the first commit.

Reply:
Sure, I thought I could squash all the commits once everything is done, or you could do it through the _Merge pull request_ option. Either way is fine with me.

Reply:
Okay, I'll squash it when merging then.

Reply:
@rami3l Never mind, bro. I squash it already. Hahaha :D